### ISSUE TITLE
FakeReplay always returning the same day

### ISSUE THREAD

**Issue by LuanG01** [Mathieu2301/TradingView-API#101]
**Describe the bug**
FakeReplay function is rarely working as Im getting returned the same day most of the time. 

**To Reproduce**
Basically just using the FakeReplay.js of the examples / test folder

**Screenshots**
![image](https://user-images.githubusercontent.com/106473877/171272491-9c8ca279-b3cf-465a-8e04-f6da2620609d.png)

Why is that and is the normal premium feature replay the same or is it worth it upgrading and using the "real" replay function instead? Would be bad if I buy premium and the bug isnt fixed with the replay function.

**Comment by Mathieu2301** [comment]
Yeah, I've seen it do that, but I don't know why... Maybe it happens when you call `fetchMore` too quickly.
The real replay mode hasn't this problem (you can test it for free with `Day`, `Week` and `Month` timeframes).
I don't know if it's worth upgrading since it's seems possible to get the `FakeReplayMode` to work.

**Comment by LuanG01** [comment]
> Sometimes it skips when the market is closed (weekends).

Im testing it on crypto so the market is never closed actually

> I don't know, it's a bit strange behavior, I've never had to deal with it.

Ye it is. I have tested it a few more times now. It seems to occur when I "spam" the replay function. After I wait a few minutes it works fine again. Really strange

**Comment by Mathieu2301** [comment]
Hmm I see but when I try it works...
<img width="129" alt="image" src="https://user-images.githubusercontent.com/21021423/171494571-4027f57f-ea02-4a6d-953e-1646c479eda2.png">
I tried with/without indicators, with normal/pro account and it works...

**Comment by LuanG01** [comment]
So strange. Its not that big of a deal for now as it works most of the time. I will give an update in case it happens more often.
